localstack/services/ses/ses_starter.py

```diff
@@ -128,6 +131,32 @@ def get_identity_verification_attributes(self):
 
     email_responses.get_identity_verification_attributes = get_identity_verification_attributes
 
+    backend_send_email_orig = SESBackend.send_email
+
+    def send_email_save_contents(self, source, subject, body, destinations, region):
+        message = backend_send_email_orig(self, source, subject, body, destinations, region)
+
+        data_dir = config.DATA_DIR
+        if data_dir == '':
+            data_dir = config.TMP_FOLDER
+
+        ses_dir = os.path.join(data_dir, 'ses')
+
+        if not os.path.exists(ses_dir):
```

Comment:
nitpick: we can use `mkdir` from `utils/common.py` here:
```
mkdir(ses_dir)
```

Reply:
Weird that my IDE did not suggest this method (I first tried by typing mkdir :P). Thanks for the hint, will be in the next update as well :)